running-locally.adoc

```diff
@@ -7,11 +7,7 @@
 [[rhmap]]
 === RHMAP
 
-The demo solution currently requires an instance of RHMAP. It can be a local or remote instance of RHMAP, but must be configured as outlined in the Raincatcher link:getting-started.adoc[Getting Started Guide] so that the local *raincatcher-demo-cloud* app can access the following RHMAP features:
-
-* Forms
-* Themes
-* Drag & Drop App Configuration
+The demo solution currently requires an instance of RHMAP. It can be a local or remote instance of RHMAP, but must be configured as outlined in the Raincatcher link:getting-started.adoc[Getting Started Guide]
```

Comment:
I asked @nialldonnellyfh about this and he said we do still need RHMAP for $fh.auth